io.asgardio.tomcat.oidc.sample/src/main/webapp/home.jsp

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<%--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  --%>
+
+<%@page import="io.asgardio.java.oidc.sdk.SSOAgentConstants" %>
+<%@page import="io.asgardio.java.oidc.sdk.bean.SessionContext" %>
+<%@ page import="io.asgardio.java.oidc.sdk.bean.User" %>
+<%@ page import="java.util.HashMap" %>
+<%@ page import="java.util.Map" %>
+
+<%
+    final HttpSession currentSession = request.getSession(false);
+    final SessionContext sessionContext = (SessionContext)
+            currentSession.getAttribute(SSOAgentConstants.SESSION_CONTEXT);
+    final String idToken = sessionContext.getIdToken();
+    
+    String name = null;
+    Map<String, Object> customClaimValueMap = new HashMap<>();
+    
+    if (idToken != null) {
+        try {
+            final User user = sessionContext.getUser();
+            customClaimValueMap = user.getAttributes();
+            name = user.getSubject();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Can we use a logger instead?

Reply:
Resolved with https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/1/commits/6b6ec648bf4aa46847e8c22e02fe4500e8a7d18e.